holidays/countries/cape_verde.py

```diff
@@ -0,0 +1,195 @@
+        "SD",  # São Domingos.
+        "SF",  # São Filipe.
+        "SO",  # São Lourenço dos Órgãos.
+        "SM",  # São Miguel.
+        "SS",  # São Salvador do Mundo.
+        "SV",  # São Vicente.
+        "TA",  # Tarrafal.
+        "TS",  # Tarrafal de São Nicolau.
+    )
+    subdivisions_aliases = {
+        # Municipality and Geographical Region.
+        "Ilhas de Barlavento": "B",
+        "Boa Vista": "BV",
+        "Brava": "BR",
+        "Santa Catarina": "CA",
+        "Santa Catarina do Fogo": "CF",
+        "Santa Cruz": "CR",
+        "Maio": "MA",
+        "Mosteiros": "MO",
+        "Paul": "PA",
+        "Porto Novo": "PN",
+        "Praia": "PR",
+        "Ribeira Brava": "RB",
+        "Ribeira Grande": "RG",
+        "Ribeira Grande de Santiago": "RS",
+        "Ilhas de Sotavento": "S",
+        "Sal": "SL",
+        "São Domingos": "SD",
+        "São Filipe": "SF",
+        "São Lourenço dos Órgãos": "SO",
+        "São Miguel": "SM",
+        "São Salvador do Mundo": "SS",
+        "São Vicente": "SV",
+        "Tarrafal": "TA",
+        "Tarrafal de São Nicolau": "TS",
+    }
+    supported_categories = (OPTIONAL, PUBLIC)
+    supported_languages = ("en_US", "pt_CV")
+
+    def __init__(self, *args, **kwargs) -> None:
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("Ano Novo"))
+
+        if self._year >= 1991:
+            # Democracy and Freedom Day.
+            self._add_holiday_jan_13(tr("Dia da Liberdade e Democracia"))
+
+        # National Heroes Day.
+        self._add_holiday_jan_20(tr("Dia da Nacionalidade e dos Heróis Nacionais"))
+
+        # Ash Wednesday.
+        self._add_ash_wednesday(tr("Quarta-feira de Cinzas"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Sexta-feira Santa"))
+
+        # Easter Sunday.
+        self._add_easter_sunday(tr("Páscoa"))
+
+        # Worker's Day.
+        self._add_labor_day(tr("Dia do Trabalhador"))
+
+        # International Children's Day.
+        self._add_childrens_day(tr("Dia Mundial da Criança"))
+
+        # Independence Day.
+        self._add_holiday_jul_5(tr("Dia da Independência Nacional"))
+
+        # Assumption Day.
+        self._add_assumption_of_mary_day(tr("Dia da Assunção"))
+
+        # All Saints' Day.
+        self._add_all_saints_day(tr("Dia de Todos os Santos"))
+
+        # Christmas Day.
+        self._add_christmas_day(tr("Natal"))
+
+    def _populate_optional_holidays(self):
+        # Holy Thursday.
+        self._add_holy_thursday(tr("Quinta-Feira Santa"))
+
+        # Mother's Day.
+        self._add_holiday_2nd_sun_of_may(tr("Dia das Mães"))
+
+        # Father's Day.
+        self._add_holiday_3rd_sun_of_jun(tr("Dia dos Pais"))
+
+    def _populate_subdiv_sv_public_holidays(self):
+        # St. Vincent Municipal Day.
```

Comment:
```suggestion
        # São Vicente Municipal Day.
```
It's municipality name, it doesn't translate.